src/ui/src/builder/BuilderFieldsAlign.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-	<div ref="rootEl" class="BuilderFieldsAlign" tabindex="-1">
+	<div
+		ref="rootEl"
+		class="BuilderFieldsAlign"
+		tabindex="-1"
+		:data-key="props.fieldKey"
```

Comment:
Is this for testing? Can we use the namespace as discussed e.g. `data-automation-key`?

Reply:
Totally forgot about this! 
done